pulp_ansible/app/viewsets.py

```diff
@@ -224,6 +290,38 @@ class CollectionVersionSignatureViewSet(NoArtifactContentUploadViewSet):
     filterset_class = SignatureFilter
     queryset = CollectionVersionSignature.objects.all()
     serializer_class = CollectionVersionSignatureSerializer
+    queryset_filtering_required_permission = "ansible.view_ansiblerepository"
```

Comment:
The parent Content viewset has special logic for figuring out what permission is needed on the repository, so this isn't needed either.
```suggestion
```